packages/forms/src/Components/CodeEditor/Enums/Language.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Filament\Forms\Components\CodeEditor\Enums;
+
+enum Language: string
+{
+    case CSS = 'css';
+
+    case HTML = 'html';
+
+    case Javascript = 'javascript';
+
+    case JSON = 'json';
+
+    case PHP = 'php';
+
+    case PlainText = 'plaintext';
```

Comment:
If it is "PlainText", should the key be `plain_text`?

```suggestion
    case PlainText = 'plain_text';
```

Reply:
@ralphjsmit Yeah it should have been. Updated.